lib/Doctrine/DBAL/Query/Expression/ExpressionBuilder.php

```diff
@@ -52,7 +52,7 @@ public function __construct(Connection $connection)
      *
      * @return CompositeExpression
      */
-    public function andX($x = null)
+    public function and($x = null)
```

Comment:
Because these are new methods, should we add variadics right now to align the definition of these methods with upcoming 3.0 or 4.0, and ease transition later?

Reply:
Definitely. They should look as close to what the consumers will get in the new versions as possible.

Reply:
Do we want the first argument to be optional here?

Reply:
> Definitely. They should look as close to what the consumers will get in the new versions as possible.

OK done!

> Do we want the first argument to be optional here?

That's the job of #3845. I'm getting lost over what to do first, and on which branch.